ES/ESum001.xml

```diff
@@ -27,7 +27,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <msIdentifier>
                   <repository ref="INS0017ʿUra"/>
                   <collection>Ethio-SPaRe</collection>
-                  <idno>UM-001</idno>
+                  <idno facs="ES/ESum001/" n="114">UM-001</idno>
```

Comment:
please style facs as we always do that, check other E-S MSS with facs inserted for the model, thank you!

Reply:
`<idno facs="UM/001/UM-001" n="051">`